CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#3346](https://github.com/thanos-io/thanos/pull/3346) Ruler UI: Fix a bug preventing the /rules endpoint from loading.
 - [#3115](https://github.com/thanos-io/thanos/pull/3115) compact: now deletes partially uploaded and blocks with deletion marks concurrently. It does that at the beginning and then every `--compact.cleanup-interval` time period. By default it is 5 minutes.
 - [#3312](https://github.com/thanos-io/thanos/pull/3312) s3: add list_objects_version config option for compatibility.
+- [#3378](https://github.com/thanos-io/thanos/pull/3378) Ruler: learned to send queries via POST. Helps when alerting/recording rules are extra long. It now uses POST unless `--query.http-method` is `GET`.
```

Comment:
learned? ;p Can we use more technical language?

Reply:
:smile: reworked this line to make it more technical and even cleaner. Let me know if that works :P